students/olszewski_bartosz/lesson_02_flow_control/The_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,12 @@
+biggest = 0
+sum_number = 0
+while True:
+    n = int(input())
+    if n > biggest:
+        sum_number = 1
+        biggest = n
+    elif biggest == n:
+        sum_number += 1
+    if n == 0:
```

Comment:
There is no reason to analyze the section above 'if n == 0' for te last 0 element so for the code readability  'if n == 0' should be placed just below 'while'

```
while True:
    n = int(input())
    if n == 0:
        break
    elif n > biggest:
        sum_number = 1
        biggest = n
    elif n == biggest:
        sum_number += 1
```